lib/app_profiler.rb

```diff
@@ -72,5 +75,9 @@ def profile_data_header
     def profile_url_formatter=(block)
       @@profile_url_formatter = block # rubocop:disable Style/ClassVars
     end
+
+    def profile_url(upload)
```

Comment:
```suggestion
    def profile_viewer_url(upload)
```
If we're going to keep this method, we should name it to distinguish the difference between a profile url and a profile viewer url and use it in `lib/app_profiler/profile.rb` too. `profile_url_formatter` should also be renamed to `profile_viewer_url_formatter`, but we would need to do a deprecation cycle for that (so maybe we could consider doing this in another change).